components/browser/menu/src/main/res/layout/mozac_browser_menu_item_image_text.xml

```diff
@@ -2,29 +2,12 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<LinearLayout
-        xmlns:android="http://schemas.android.com/apk/res/android"
-        xmlns:tools="http://schemas.android.com/tools"
-        style="@style/Mozac.Browser.Menu.Item.Container"
-        android:layout_width="match_parent"
-        android:orientation="horizontal"
-        android:clickable="true"
-        android:gravity="center_vertical"
-        android:focusable="true">
-
-    <androidx.appcompat.widget.AppCompatImageView
-            android:id="@+id/image"
-            style="@style/Mozac.Browser.Menu.Item.ImageText.Icon"
-            android:clickable="false"
-            android:background="@android:color/transparent"
-            tools:src="@android:drawable/screen_background_dark"/>
-
-    <TextView
-            android:id="@+id/text"
-            style="@style/Mozac.Browser.Menu.Item.ImageText.Label"
-            android:clickable="false"
-            android:background="@android:color/transparent"
-            android:gravity="center_vertical"
-            tools:text="Item"
-            android:importantForAccessibility="no"/>
-</LinearLayout>
\ No newline at end of file
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/imageText"
```

Comment:
Would be nice to use `tools:drawableStart` and `tools:text` so the design view has some sample data.